bgpd/bgp_zebra.c

```diff
@@ -2705,6 +2706,35 @@ static int bgp_pbr_get_ifnumber(struct bgp *bgp)
 	return cnt;
 }
 
+struct egress_data {
```

Comment:
struct egress_data is redefined  , recommend define it once in a header file

Reply:
Ack